src/metabase/automagic_dashboards/core.clj

```diff
@@ -378,39 +384,46 @@
                  (str/starts-with? name "id_")
                  (str/ends-with? name "_id"))))))
 
+(defn- field-spec-pred
+  "Generate a predicate of the form f(field) -> true | false based on a field spec."
```

Comment:
`f(field)` is a weird way to write a function signature for a lisp

Reply:
I was thinking "mathy" but I'll change to `(f field)`.